src/lib/y2storage/filesystems/mount_by_type.rb

```diff
@@ -90,6 +90,14 @@ def udev_name(value)
       end
 
       class << self
+
+        alias_method :all_nonfiltered, :all
```

Comment:
Inconsistent use of `nonfiltered` vs `unfiltered` (below, in the `MountPoint` class).

Reply:
and which one you prefer?

Reply:
Well, actually what we always do when we override a list or value coming from libstorage-ng is having `storage_whatever` for the raw value and then `whatever` for the converted one.

For consistency I guess we should have `MountByType.storage_all` vs `MountByType.all` and `MountPoint#storage_possible_mount_bys` vs `MountPoint@possible_mount_bys`.